### ISSUE TITLE
Missing reference for all standard libraries

### ISSUE THREAD

**Issue by stunlocked1** [julia-vscode/julia-vscode#3532]
Those libraries work, but VSCode doesn't see them

![image](https://github.com/julia-vscode/julia-vscode/assets/76593873/a603e3da-ed11-421f-9932-a9a5a6d1e5a5)

And I tried reinstalling both Julia and VSCode and that didn't work.

**Comment by j-adel** [comment]
Yes I also get the same warnings even though Julia runs it just fine. The only fix I found (besides turning off the warning) is to create a project environment where the packages are added. Of course Julia doesn't need this, and it runs just fine without a project.toml because it just uses the global variable. The linter should recognize the global environment in the absence of a project.toml.

**Comment by davidanthoff** [comment]
Closing: this was fixed in v1.220.0 — the standard-library symbol store is now shared across environments, and the new `UnresolvedImport` diagnostic replaced the blanket missing-reference errors that used to appear for unindexed names.

Please reopen or file a new issue if you still see it on a current release.
